wiki/Shortcut_key_reference/en.md

```diff
@@ -194,32 +194,32 @@ These shortcuts work anywhere within the beatmap editor:
 
 #### Mouse
 
-| Shortcut | Description |
-| --- | --- |
-| `Right Click` | Toggle new combo (when placing new hit objects) or delete object (when selecting) |
+| Shortcut | Action |
+| :-- | :-- |
+| `Right Click` | Toggle new combo (when placing new hit objects) or delete object (when selecting). |
 
 #### General
 
-| Shortcut | Description |
-| --- | --- |
+| Shortcut | Action |
+| :-- | :-- |
 | `Alt` + `F4` or `Esc` | Return to song selection. |
-| `Ctrl` + `Z` | Undo |
-| `Ctrl` + `Y` | Redo |
+| `Ctrl` + `Z` | Undo. |
```

Comment:
I left the full stops out since they weren't really full sentences imo.

Reply:
well... technically they are, I see what u mean though. I just put them on to be consistent with everything else, I'm fine with either